devices/MAX1704x/Max17048.cs

```diff
@@ -0,0 +1,453 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.MAX1704x
+{
+    /// <summary>
+    /// Represents the MAX17048 class, which inherits from MAX17044.
+    /// </summary>
+    public class Max17048 : Max17044
+    {
+        /// <summary>
+        /// <inheritdoc />
+        /// </summary>
+        protected override float FullScale => 5.12f;
+
+        /// <summary>
+        /// <inheritdoc />
+        /// </summary>
+        protected override float Divider => 65536.0f;
+
+        /// <summary>
+        /// Gets or sets the reset voltage of a device.
+        /// </summary>
+        /// <exception cref="ArgumentOutOfRangeException">Thrown when the provided value is outside the valid range of 0 to 5.08.</exception>
+        public float ResetVoltage
```

Comment:
We're using UnitsNet for Voltage, Ampers, etc.

Reply:
done